modules/basics/pages/system-and-language-basics.adoc

```diff
@@ -65,6 +67,50 @@ To exit the GSQL shell, type either `exit` or  `quit` at the GSQL prompt:
 
 `GSQL> EXIT`  or `GSQL> QUIT`
 
+=== New token-based authentication method
```

Comment:
Never saw "New" (unless it's the Release Notes, which is an explicitly version-specific document).